[2020-05-14 09:09:53,241] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-14 09:16:17,766] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 10:49:55,918] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 10:50:16,498] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 12:26:34,700] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 12:42:04,212] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 12:46:12,078] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 12:47:47,532] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 12:49:29,356] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 12:49:49,973] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 12:52:15,658] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 12:53:19,726] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 12:53:43,348] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 15:30:26,947] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 15:31:50,769] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 15:35:30,507] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-15 15:57:18,458] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 07:37:50,759] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 07:40:15,870] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 07:42:11,883] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 07:46:21,002] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 07:49:02,006] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 07:52:11,569] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 07:53:19,131] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 08:13:22,529] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 08:14:06,466] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 08:14:39,302] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 08:15:11,893] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 08:15:39,151] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 08:16:22,989] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 08:19:59,055] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 08:20:36,107] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 15:55:04,071] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 16:01:01,799] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 16:03:00,821] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 16:06:10,277] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 16:10:02,757] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 16:15:22,379] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 16:19:36,609] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 16:23:02,156] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 16:36:44,066] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 16:39:33,562] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 16:41:33,439] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 16:43:56,638] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 16:58:32,650] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 17:00:12,070] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 19:45:10,132] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 20:20:42,982] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-18 20:26:53,074] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
